packages/shared/src/const.ts

```diff
@@ -18,6 +18,9 @@ export const Env = {
   /** Public URL base that tiles are served from */
   PublicUrlBase: 'BASEMAPS_PUBLIC_URL',
 
+  /** Location of sprites and glyphs */
+  AssetLocation: 'BASEMAPS_ASSEST_LOCATION',
```

Comment:
Do we need to setup new environment variable in the lambda stack?

Reply:
We will, I am wanting to use this as a way to bundle the sprites and glyphs into the lambda.zip when we deploy a entire package of it.

So some time in the future we should have basemaps serving fonts/glyphs from `/v1/...` rather than the directly from the static bucket.